packages/interop-test-client/src/index.ts

```diff
@@ -0,0 +1,258 @@
+// This script implements the client role of draft-dcook-ppm-dap-interop-test-design-01.
```

Comment:
nit: does this actually change anything or is it just moved from `scripts/interopTestClient.ts`? `git mv` might help git track the rename and preserve the file's history.

Reply:
I changed the indentation to match the rest of the repository, which is what's keeping it from recognizing the move. Here's a diff ignoring whitespace.

```diff
diff --git a/scripts/interopTestClient.ts b/after.ts
index b873223..fe72521 100644
--- a/scripts/interopTestClient.ts
+++ b/after.ts
@@ -1,6 +1,6 @@
 // This script implements the client role of draft-dcook-ppm-dap-interop-test-design-01.
 import express, { Request, Response } from "express";
-import DAPClient from "dap";
+import DAPClient from "@divviup/dap";
 
 interface CountVdafObject {
   type: "Prio3Aes128Count",
@@ -241,6 +241,7 @@ async function uploadHandler(req: Request, res: Response): Promise<void> {
   }
 }
 
+export function interopTestClient() {
   const app = express();
   app.use(express.json());
 
@@ -254,3 +255,4 @@ app.post("/internal/test/upload", (req, res, next) => {
 
   console.debug("Starting server on port 8080")
   app.listen(8080);
+}
```

Reply:
Ah, that makes sense, and I don't think it's worth the hassle of tricking git(Hub) into recognizing the move.